bicep/network-new.bicep

```diff
@@ -273,217 +187,15 @@ var nsg_rules = {
 
     // Deny all remaining traffic and allow Internet access

```

Comment:
This comment is out of date, as we are commenting out the DenyVnetOutbound now.

Reply:
I will review all rules